### PR TITLE
Start adding new Ads CSP domains

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves https://github.com/bbc/simorgh-infrastructure/issues/1300

**Overall change:**
After as recent audit of CSP failures on the WS sites with Adverts GNL have come up with a new list of domains they would like us to add to our CSP allow list.

```
encrypted-tbn*.gstatic.com - This is not being added for now.
securepubads.g.doubleclick.net & bbc.gscontxt.net are moved to script-src.
```

**Code changes:**
Update the cspHeader file with the new domains.
Create new directives for prefetch-src

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by chris-hinds — COMMENTED** [review]
👍  cant approved as I pushed the first commit

File: src/server/utilities/cspHeader/index.test.js [hunk 1]
```diff
@@ -536,6 +582,8 @@ describe('cspHeader', () => {
         'https://tpc.googlesyndication.com',
         'https://gn-web-assets.api.bbc.com',
         'https://www.googletagservices.com',
+        'https://securepubads.g.doubleclick.net',
+        'https://bbc.gscontxt.net',
```
Comment (joshcoventry) [inline src/server/utilities/cspHeader/index.test.js:584]: This seems to be a duplication of line 570?

File: src/server/utilities/cspHeader/index.test.js [hunk 2]
```diff
@@ -254,6 +276,8 @@ describe('cspHeader', () => {
...
```
Comment (joshcoventry) [inline src/server/utilities/cspHeader/index.test.js:279]: Seems to be a duplication of line 264?

File: src/server/utilities/cspHeader/index.js [hunk 3]
```diff
@@ -63,7 +73,10 @@ const advertisingDirectives = {
     'https://tpc.googlesyndication.com',
     'https://gn-web-assets.api.bbc.com',
     'https://www.googletagservices.com',
+    'https://securepubads.g.doubleclick.net',
+    'https://bbc.gscontxt.net',
```
Comment (joshcoventry) [inline src/server/utilities/cspHeader/index.js:76]: Duplication of line 61?

File: src/server/utilities/cspHeader/index.test.js (hunk 1 above)
Comment (chewu) [reply src/server/utilities/cspHeader/index.test.js:584]: This is for testing: isAmp: false, isLive: false

File: src/server/utilities/cspHeader/index.test.js (hunk 2 above)
Comment (chewu) [reply src/server/utilities/cspHeader/index.test.js:279]: This is for testing: isAmp: false, sLive: true

File: src/server/utilities/cspHeader/index.js (hunk 3 above)
Comment (chewu) [reply src/server/utilities/cspHeader/index.js:76]: Good spot, removed duplication.

**Review by joshcoventry — APPROVED** [review]

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**